src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -5,6 +5,7 @@
 
 'use strict';
 
+import * as path from 'path';
```

Comment:
@sbatten layer breaker, you cannot have a node.js dependency in `browser` code. there is a `vs/base/path` library you can use instead.